pyproject.toml

```diff
@@ -14,20 +17,26 @@ log_cli_level = "INFO"
 line-length = 99
 target-version = ["py38"]
 
-[tool.isort]
-line_length = 99
-profile = "black"
-
 # Linting tools configuration
-[tool.flake8]
-max-line-length = 99
-max-doc-length = 99
+[tool.ruff]
+line-length = 99
+select = ["E", "W", "F", "C", "N", "D", "I001"]
+extend-ignore = [
+    "D203",
+    "D204",
+    "D213",
+    "D215",
+    "D400",
+    "D404",
+    "D406",
+    "D407",
+    "D408",
+    "D409",
+    "D413",
```

Comment:
Please explain the reasoning for disabling these rules.

Reply:
These rules all refer to formatting rules for sections within docstrings. The coding styles for the docstring pieces are more in-line with the google conventions rather than numpy or strict pep257 conventions ([reference here](http://www.pydocstyle.org/en/stable/error_codes.html#default-conventions)). This is a docstring convention standard more than anything and should align with existing standards.